pkg/agent/cniserver/sriov.go

```diff
@@ -51,6 +49,10 @@ func GetPodContainerDeviceIDs(podName string, podNamespace string) ([]string, er
 	ctx, cancel := context.WithTimeout(context.Background(), connectionTimeout)
 	defer cancel()
 
+	if runtime.IsWindowsPlatform() {
```

Comment:
Reminder.

Reply:
need further clarification on this @jianjuns. I had to add this due to golangci-fix errors during initial PR submission and the func call was not protected by featuregate. Not sure I have some misunderstanding on the unsupportedFeatureOnWindows usage. But, I will discuss with you this evening over call and take care accordingly.

Reply:
+1 to Jianjun's comment. Now that you have updated `unsupportedFeatureOnWindows`, it is not possible for this code to be executed at runtime on Windows, as the agent would fail earlier with a fatal error.